src/otx/algo/detection/mmdeploy/atss.py

```diff
@@ -0,0 +1,11 @@
+"""MMDeploy config of ATSS model for Detection Task."""
```

Comment:
```suggestion
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
#
"""MMDeploy config of ATSS model for Detection Task."""
```

Could you please add the license to your files? many .py files located at the mmdeploy folder missed the license

Reply:
This is copy from mmdeploy repository. Is it ok to add our copyright?

Reply:
@yunchu  could you check this ? How can we deal with this case?

Reply:
if you copied the contents from the other opensource, it's better to keep the original header and recommended to add some commenst about the origination. I think this [file](https://github.com/openvinotoolkit/training_extensions/blob/7c3617ea3427c497375f80e945ff56984da45ef2/src/otx/algorithms/visual_prompting/adapters/pytorch_lightning/models/visual_prompters/segment_anything.py) would be the example of that.

Reply:
Thanks a lot :)

Reply:
I checked mmdeploy file but there is no copyright in config file. I'll just add reference in each file.

Reply:
done at https://github.com/openvinotoolkit/training_extensions/pull/2806/commits/7a6ac90ea226e9c1eb06f86fdba52d81c4f13f76